### DISCUSSION TITLE
Remove recipe

### DISCUSSION THREAD

**Discussion by blowk** [TandoorRecipes/recipes#2665] [General]
I've just started with Tandoor Recipes.
I added some recipes, but now I want to remove some 'unwanted' recipes but I cannot find a remove button. Only the option to change the content.
Can anybody help me how to remove recipes?

**Comment by smilerz** [comment]
"Delete" is available while editing a recipe.

**Reply by blowk** [reply]
Thanks, I didn't look further.
Stupid that I didn't noticed the button below.
